sf-fx-runtime-java-runtime/src/main/java/com/salesforce/functions/jvm/runtime/invocation/undertow/UndertowInvocationInterface.java

```diff
@@ -51,15 +52,28 @@ public UndertowInvocationInterface(int port, String host) {
   @Override
   public void start(
       ProjectFunction<CloudEvent, SalesforceFunctionResult, SalesforceFunctionException>
-          projectFunction)
-      throws Exception {
-    Undertow undertow =
+          projectFunction) {
+    this.undertow =
         Undertow.builder()
             .addHttpListener(port, host)
             .setHandler(new ProjectFunctionHandler(projectFunction))
             .build();
 
     undertow.start();
+  }
+
+  @Override
+  public void stop() throws Exception {
+    undertow.stop();
+  }
+
+  @Override
+  public boolean isStarted() {
+    return this.undertow != null;
```

Comment:
It looks like `isStarted` will still return true after `stop()` has been called. Is this expected? (I can't tell if `isStarted` is meant to be "has been started at some point" or "is running" etc).

Reply:
Great catch! I will fix this and add tests! 👍🏻

Reply:
Fixed in 8e29954